devtools/maven/pom.xml

```diff
@@ -190,6 +249,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-plugin-plugin</artifactId>
+                <version>3.7.1</version>
```

Comment:
I'd just generally move for whole build to 3.8.1 version. The POMs build-parent, bootstrap-parent and extension-maven-plugin needs this added to pluginMgmt